config/mongoid.yml

```diff
@@ -1,19 +1,18 @@
 development:
   sessions:
     default:
-      database: <%= ENV['MONGO_DB_NAME'] || 'content_store_development' %>
-      hosts:
-        - localhost:27017
+      # Contains draft_content_store_development or content_store_development
+      # depending on whether we're running content store in draft mode or not.
+      uri: <%= ENV['MONGODB_URI'] %>
```

Comment:
It's worth having a fallback value here so that it's still possible to run this in dev mode without govuk_setenv.

Also, the comment above doesn't seem quite correct, as this contains the full URI, not just the database name.